src/FactorySql.re

```diff
@@ -0,0 +1,60 @@
+/* Private */
+let removeHeadOfList = list =>
+  switch (List.tl @@ list) {
+  | exception _ => []
+  | result => result
+  };
+
+let mergeGroupBy = (base, userClauses) =>
+  SqlComposer.Select.(
```

Comment:
don't open this module here to use it just once! makes it slightly harder to read.

Reply:
Fragment from refactoring, I will fix it in a future PR.